purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/trustedentitlements/TrustedEntitlementsInformationalModeIntegrationTest.kt

```diff
@@ -79,56 +68,22 @@ class TrustedEntitlementsInformationalModeIntegrationTest : BasePurchasesIntegra
     }
 
     @Test
-    fun verificationChangesAfterSuccessIsNotified() {
-        var receivedCustomerInfo: CustomerInfo? = null
-        ensureBlockFinishes { latch ->
-            Purchases.sharedInstance.getCustomerInfoWith(
-                onError = { fail("should be success. Error: ${it.message}") },
-                onSuccess = {
-                    receivedCustomerInfo = it
-                    latch.countDown()
-                },
-            )
-        }
-
-        assertThat(receivedCustomerInfo).isNotNull
-        assertThat(receivedCustomerInfo?.entitlements?.verification).isEqualTo(VerificationResult.VERIFIED)
-
+    fun verificationChangesAfterSuccessIsNotified() = runTestActivityLifecycleScope {
+        val receivedCustomerInfo = Purchases.sharedInstance.awaitCustomerInfo()
+        assertThat(receivedCustomerInfo.entitlements.verification).isEqualTo(VerificationResult.VERIFIED)
         Purchases.sharedInstance.forceSigningErrors = true
 
-        var receivedCustomerInfo2: CustomerInfo? = null
-        ensureBlockFinishes { latch ->
-            Purchases.sharedInstance.getCustomerInfoWith(
-                fetchPolicy = CacheFetchPolicy.FETCH_CURRENT,
-                onError = { fail("should be success. Error: ${it.message}") },
-                onSuccess = {
-                    receivedCustomerInfo2 = it
-                    latch.countDown()
-                },
-            )
-        }
+        val receivedCustomerInfo2 = Purchases.sharedInstance.awaitCustomerInfo()
```

Comment:
Hmm this is not using the `FETCH_CURRENT` cache fetch policy. Looks like we didn't add that parameter to the coroutines version. I would think this test would fail, since it should be getting the customer info from cache, which is actually verified.

I think we can add that missing parameter in a separate PR and then rebase this off that.

Reply:
> Will cut a separate PR

There you go https://github.com/RevenueCat/purchases-android/pull/1086

Reply:
> update this one afterwards

I've just rebased from `main` and added `CacheFetchPolicy.FETCH_CURRENT` to `verificationChangesAfterSuccessIsNotified`. Wondering though why the test was passing without it 🤔